src/components/Avatar.js

```diff
@@ -64,12 +64,14 @@ class Avatar extends PureComponent {
             <View pointerEvents="none" style={this.props.containerStyles}>
                 {_.isFunction(this.props.source) || this.state.imageError
                     ? (
-                        <Icon
-                            src={this.state.imageError ? this.props.fallbackIcon : this.props.source}
-                            height={iconSize}
-                            width={iconSize}
-                            fill={this.state.imageError ? themeColors.offline : this.props.fill}
-                        />
+                        <View style={this.props.imageStyles}>
```

Comment:
Shouldn't we use `imageStyle` here instead of the prop? Then check my note https://github.com/Expensify/App/issues/12645#issuecomment-1319059138 about a issue.

Reply:
GetAvatarStyle passes the background color which is themeColors.offline

If you notice the Icon component already has that applied
fill: this is the background color and it will be set to the same value of themeColors.offline

Reply:
So what are you suggesting to do here?  Do not apply `imageStyle` as style and keep this logic.

But will be safe?  
`imageStyle` also has borderRadius which won't be applied to the new View you added.

Reply:
I think its okay to keep it as it is. The View will only span the size of its content by default. And the border radius wont affect it even if applied unless we give it a border. 

If you feel strongly about it i can still change it.

Reply:
> And the border-radius won't affect it even if applied unless we give it a border.

I doubt that. 

To test:
1. To any avatar, let's  pass a background color via imageStyles prop. 
Then you will see a sqaure are covered with the color. 
![image](https://user-images.githubusercontent.com/24370807/203438398-100ed549-9b80-4c54-86b1-5dad8a83136f.png)


--- 
Instead when you apply `imageStyle`. 

It will be circle 
![image](https://user-images.githubusercontent.com/24370807/203438533-2562be9f-d659-49bd-8ab3-3e17b04dba94.png)

Reply:
@parasharrajat Thank you so much for correcting me, you were correct there. Updated the code.

I also ran a test on Web/ Web Safari and Web Chrome. all of them still look the same.